protobuf/v1/registration.proto

```diff
@@ -13,6 +13,14 @@ message RegisterRequest {
   string grpc_endpoint = 2;
   // uuid generated by the instance on startup
   google.protobuf.StringValue instance_uuid = 3;
+  // latest api version supported by the dataplane
+  APIVersion api_version = 4;
+}
+
+// latest api version supported by the dataplane
+enum APIVersion {
```

Comment:
Would it be worth reporting a list of versions?

Reply:
Done.